site/docs/v1/tech/reference/configuration.adoc

```diff
@@ -296,6 +296,9 @@ Deprecated environment variable names:
 +
 * `FUSIONAUTH_KICKSTART`
 
+[field]#fusionauth-app.local-metrics.enabled# [type]#[Boolean]# [default]#defaults to `false`# [since]#Available since 1.46.0#::
+Set to `true` to allow unauthenticated access to `/api/prometheus/metrics` and `/api/status` from `localhost`. This can be used to to allow the scraping of health and metrics data from local collecter agents without the need to provision an API key.
```

Comment:
```suggestion
Set to `true` to allow unauthenticated access to `/api/prometheus/metrics` and `/api/status` from `localhost`. This can be used to allow the scraping of health and metrics data by local collector agents without the need to provide an API key.
```